mhapsite/mhap/models.py

```diff
@@ -45,6 +45,7 @@ def get_list_url(self):
 #http://stackoverflow.com/questions/1737017/django-auto-now-and-auto-now-add
 class ChatMessages(models.Model):
     message = models.CharField(max_length=255)
+    message_type = models.CharField(max_length=255, default="default")
```

Comment:
Instead of having models.CharField

have 
````python
    QUOTE = 'QE'
    HELP = 'HP'
    DEFAULT = 'DT'

BOT_CHOICES=(
(QUOTE,"QUOTE"),
(HELP ,"HELP"),
(DEFAULT ,"DEFAULT"),
)

    message_type = models.CharField(
        max_length=2,
        choices=BOT_CHOICES,
        default=DEFAULT,
    )
````